src/consume.ts

```diff
@@ -36,22 +45,71 @@ export function activateMessageViewer(context: ExtensionContext) {
   rate limiting for multiple active message viewers. */
   const schedule = scheduler(4, 500);
 
+  /* We track active topic as a kafka topic of a webview panel that is currently
+  visible on the screen. When the user clicks on Duplicate Message Browser action
+  at the top of the window, we can use the topic entity to start another message
+  viewer with the same topic. Otherwise, we use webview panel cache to only keep
+  a single active message browser per topic. */
+  let activeTopic: KafkaTopic | null = null;
```

Comment:
Nice, I was wondering how this behaved with the command available in the command palette 👍

Reply:
The duplicate command isn't available in the command palette. I added `enablement` to prevent it. I'm just not sure if there's much use of it in the command palette since it highly depends on the execution context.

Reply:
It is, but only if you have message viewer open already:
<img width="970" alt="image" src="https://github.com/user-attachments/assets/fbb68dfe-3e87-466d-adf0-1dcf60452141">

May need a specific `when: false` in the `commandPalette` section of `contributes.menu` to prevent this entirely, but it seemed to work fine as far as I could tell, and didn't appear when you had a tab open that wasn't message viewer

Reply:
> It is, but only if you have message viewer open already:

Oh, so maybe still convenient then? 😅